solver/llbsolver/file/backend.go

```diff
@@ -204,6 +225,10 @@ func cleanPath(s string) string {
 	return s2
 }
 
+func rootPath(root, src string) (string, error) {
```

Comment:
I thought a function that also calls rm and does the `action.AllowNotFound` check.

Reply:
When `AllowWildcard` is true and is removing each of the matched paths, I don't think `AllowNotFound` makes sense there?

Reply:
It should be a no-op for that case, assuming that directory is immutable. If it is not behavior can be undefined.

Reply:
If you want to keep the different `AllowWildcard` behavior, feel free to revert this as a single line helper function doesn't really improve this code, just makes is more complicated to read through.

Reply:
I refactored it as suggested :+1: